build.sbt

```diff
@@ -50,15 +30,15 @@ val sonatypeReleaseSettings = Seq(
 
 def projectWithPlayVersion(majorMinorVersion: String) =
   Project(s"play-v$majorMinorVersion", file(s"play-v$majorMinorVersion")).settings(
-    scalaVersion       := "2.12.12",
-    crossScalaVersions := Seq(scalaVersion.value, "2.13.4"),
+    scalaVersion       := "2.12.15",
+    crossScalaVersions := Seq(scalaVersion.value, "2.13.7"),
     scalacOptions ++= Seq("-feature", "-deprecation"),
 
     libraryDependencies ++= Seq(
-      "com.gu.play-secret-rotation" %% "core" % "0.18",
-      "org.typelevel" %% "cats-core" % "2.0.0",
+      "com.gu.play-secret-rotation" %% "core" % "0.31",
```

Comment:
Note this pulls in a new version of scaffeine ([v5.1.1](https://github.com/blemale/scaffeine/releases/tag/v5.1.1)), and you may get runtime errors unless you upgrade to a sufficiently recent version.